indico/htdocs/sass/modules/_papers.scss

```diff
@@ -15,7 +15,9 @@
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
 
-@import base;
+@import 'base';
```

Comment:
You don't need any of these imports anymore. You needed them before because we were not bundling module-related `scss` together with `screen.scss`, but that's not the case anymore. We now have all variables and partials available out-of-the-box in module-related `scss`.